source/simulator/helper_functions.cc

```diff
@@ -1120,7 +1120,10 @@ namespace aspect
           for (unsigned int q=0; q<output.size(); ++q)
             {
               Tensor<1,dim> g = gravity_->gravity_vector(in.position[q]);
-              output[q] = std::fabs(std::min(0.0, g*velocity_values[q]/g.norm()))*year_in_seconds;
+              if (g.norm() < 2.0*std::numeric_limits<double>::min())
+                output[q] = 0.0;
+              else
+                output[q] = std::fabs(std::min(0.0, g*velocity_values[q]/g.norm()))*year_in_seconds;
```

Comment:
The original line produced a divide by zero exception when using the depth average postprocessor if g in a parameter file was equal to zero. This is what we do in the fixed strain rate test, but I'm sure there are other prm files out there with g=0 which would also fail if we tried to use the depth average postprocessor. 

What do you think?